postorius/Dockerfile

```diff
@@ -12,16 +12,18 @@ COPY docker-entrypoint.sh /usr/local/bin/
 # rights for management script
 RUN set -ex \
 	&& apk add --no-cache --virtual .build-deps gcc libc-dev linux-headers \
-	   	postgresql-dev mariadb-dev libffi-dev \
-	&& apk add --no-cache --virtual .mailman-rundeps bash libffi \
-	   postgresql-client mysql-client py-mysqldb curl mailcap \
-	&& pip install -U 'Django<3.0' pip \
-	&& pip install postorius==1.3.1 \
+		postgresql-dev mariadb-dev python3-dev libffi-dev \
+	&& apk add --no-cache --virtual .mailman-rundeps bash sassc \
+		postgresql-client mysql-client py-mysqldb curl mailcap \
+		python3 py3-setuptools libffi \
+	&& python3 -m pip install -U 'Django<3.0' pip \
+	&& python3 -mpip install postorius==1.3.1 \
 		uwsgi \
 		psycopg2 \
 		dj-database-url \
 		mysqlclient \
 		typing \
+		django_compressor \
```

Comment:
This IMO might be un-necessary. The current version does not need `django-compressor` since it is a dependency of Hyperkitty.
I have added a comment on #359 regarding the same.

Reply:
As per my tests, it causes an error if removed.

Reply:
I looked at it, it looks like we should remove `'compressor.finders.CompressorFinder',`  from the `STATICFILES_FINDERS`.

https://github.com/maxking/docker-mailman/blob/master/postorius/mailman-web/settings.py#L197

Reply:
Tested - seems working! PR has updated